cmd/buildah/common.go

```diff
@@ -224,6 +226,9 @@ func getImageConfig(ctx context.Context, sc *types.SystemContext, store storage.
 	}
 	image, err := ref.NewImage(ctx, sc)
 	if err != nil {
+		if img, err2 := store.Image(imageID); err2 == nil && img.ID == imageID {
```

Comment:
I take it that there is no "ImageAlreadyExists" err code?

Reply:
The storage library can return errors when a caller tries to create an image names or an ID that are specified for the new image are already in use, yes, but here we're trying to avoid fatally breaking `getParent()` and `getChildren()`, which would otherwise break `imageIsParent()`, which would in turn produce an error when `buildah images` tried to look at an image that doesn't have a configuration blob, which is not an error for any image which exists solely to hold a manifest list.